services/QuillLMS/spec/queries/snapshots/most_active_schools_query_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Snapshots
+  describe MostActiveSchoolsQuery do
+    include_context 'Snapshots Period CTE'
+
+    context 'big_query_snapshot', :big_query_snapshot do
+      let(:num_classrooms) { 11 }
+
+      let(:classroom_units) { classrooms.map { |classroom| create(:classroom_unit, classroom: classroom) } }
+
+      # For each classroom (each of which has a single classroom_unit), create activity_sessions for it, but creating one less for each subsequent classroom so that they'll have different relevant counts
+      let(:activity_sessions) do
+        classroom_units.map.with_index do |classroom_unit, i|
+          create_list(:activity_session, (num_classrooms - i), classroom_unit: classroom_unit)
+        end
+      end
+
+      let(:runner_context) {
+        [
+          classrooms,
+          teachers,
+          classrooms_teachers,
+          schools,
+          schools_users,
+          classroom_units
+        ]
+      }
+
+      let(:runner) { QuillBigQuery::TestRunner.new(cte_records) }
```

Comment:
`let(:runner)` and `let(:results)` can be removed since they are already defined in the shared_context [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/spec/support/shared_contexts/quill_big_query_test_runner_setup.rb#L7)